doc/HOWTO_oracle.md

```diff
@@ -57,92 +76,79 @@ sudo ln -s libocci.dylib.12.1   libocci.dylib
+sqlplus "sys/Oradoc_db1@(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=localhost)(PORT=1521))(CONNECT_DATA=(SERVER=DEDICATED)(SERVICE_NAME=ORCLCDB.localdomain)))" as sysdba
 ```
 
-* Verify you see an output like this:
+Your output should show something like this:
 
 ```
-SQL*Plus: Release 12.1.0.2.0 Production on Fri Mar 3 13:48:51 2017
+Connected to:
+Oracle Database 12c Enterprise Edition Release 12.2.0.1.0 - 64bit Production
 
-Copyright (c) 1982, 2016, Oracle.  All rights reserved.
-
-ERROR:
-ORA-12154: TNS:could not resolve the connect identifier specified
-
-
-Enter user-name:
+SQL> 
 ```
 
-* `^D` out of that and carry on.
-
 ## Setting up the Database
 
-##### Install Required Gems
-
-* Make sure you have the `activerecord-oracle_enhanced-adapter` and `ruby-oci8` gems
-enabled.
-
-##### Set up database configuration
-
-* Copy from oracle settings template
-
-```
-cp config/database.yml.oracle.template config/database.yml
-```
-
-* Validate keys in settings are correct for your current application
-
-* Start, setup, and seed the oracle container:
-
-```
-docker/oracle/setup.sh
-# demo:seed is optional
-rake demo:seed
-```
+1. Make sure you have the `activerecord-oracle_enhanced-adapter` and `ruby-oci8` gems enabled in your `Gemfile`.
+1. Copy the Oracle template for `database.yml` by running `cp config/database.yml.oracle.template config/database.yml`.
+1. Run `bundle exec rake db:setup`.
+1. Optionally, run `bundle exec rake demo:seed` if you want to populate the database with demo data.
 
 # Optional Extras
 
-##### Install Oracle SQL Developer
+## Install Oracle SQL Developer
 
 * Download from: `http://www.oracle.com/technetwork/developer-tools/sql-developer/overview/index.html`
 
 * Install into `/Applications`
 
-#### Restore from a backup
+## Restore From Backup
 
 Run `bundle exec rake db:oracle_drop_severe`. This will ensure that your database
 is clean. Without it the import might skip tables due to them already existing.
 
 Assuming you used `$HOME/oracle_data` as the volume location when you did `docker run`:
 
-Copy the `.dmp` file to `$HOME/oracle_data/admin/xe/dpdump/`
+1. Copy the `.dmp` file to `$ORACLE_DATA_DIR/u01/app/oracle/admin/ORCL/dpdump/` (assuming you set `ORACLE_DATA_DIR` above), so it is located in the server’s default data pump directory.
 
-Get a bash shell inside your container:
+1. Start a bash shell in the `oracle` container:
 
-```
-docker exec -it nucore_db bash
-```
+    ```
+    docker exec \
+      --interactive \
+      --tty \
+      oracle \
+      bash
+    ```
+
+1. Import the dump, replacing DUMPFILE with the name of your dump file, and REMAP_SCHEMA with your database’s username if necessary:
 
-Run this (replacing the DUMPFILE filename and the second part of REMAP_SCHEMA with your database's username):
 ```
-impdp system/oracle@//localhost:1521/xe DIRECTORY=data_pump_dir DUMPFILE=expdp_schema_COR1PRD_201708191913.dmp REMAP_SCHEMA=bc_nucore:nucore_nu_development
+impdp \
```

Comment:
This step didn't work for me. It's telling me `TNS:listener does not currently know of service requested in connect descriptor`

Reply:
f6a5913 should fix this, along with subsequent permissions errors because even though the image mounts at `/ORCL`, the server’s `DATA_PUMP_DIR` expects files in `/u01` 🤦‍♂️